Sources/App/routes.swift

```diff
@@ -11,8 +11,29 @@ public func routes(_ router: Router) throws {
     router.get("hello") { req in
         return "Hello, world!"
     }
+    
+    router.post(GithubPullRequestWebhook.self, at: "pullRequestActivity") { (req, webhook) -> Future<Response> in
```

Comment:
To be sure, this just saves the performance of a merged PR to disk, right?

Reply:
The flow is
1. PR is merged
2. `pullRequestActivity` route is triggered by github webhook
3. `performance-test` job is started, and job number is saved to cache
4. Job finished 
5. CircleCI triggers the `circle` route via webhook
6. The triggering job number is extracted from the webhook's payload, and the cache is checked for the existence of the job number
7. If the job number exists in the cache, we know that this job was created as the result of a merge into master, and we need to save the results to whatever DB we're using
8. Results are saved, and used as the baseline during future performance tests